src/main/java/controller/home/Equipment_lvup.java

```diff
@@ -0,0 +1,88 @@
+package controller.home;
+
+import model.Status;
+import model.Equipment;
+import model.EquipmentItem;
+import model.util.User;
+
+public class Equipment_lvup {//装備を作成完了すぐに後に実行したい
+    public Equipment lvup(User user ,Equipment equipment) {
+        if(user.equipmentPossesion(equipment)) {//装備を持ってい場合実行
+            switch (equipment.item.name) {//装備名で上げ幅を決定
+                case "木の剣":
+                case "木の槍":
+                case "木の弓":
+                case "木の短剣":
```

Comment:
```suggestion
            switch (equipment.item) {//装備名で上げ幅を決定
                case WOOD_SWORD:
                case WOOD_SPEAR:
                case WOOD_ARROW:
                case WOOD_DAGGER:
```
こんな感じでEnumでswitchする形にお願いしたいです